doc/source/extended_documentation/calibration/reliability_calibration/reliability_calibration_examples.rst

```diff
@@ -0,0 +1,27 @@
+**Examples**
+
+The reliability calibration tables returned by this plugin are structured as shown below::
+
+  reliability_calibration_table / (1) (air_temperature: 2; table_row_index: 3; probability_bin: 5; projection_y_coordinate: 970; projection_x_coordinate: 1042)
+       Dimension coordinates:
+            air_temperature                           x                   -                   -                           -                             -
+            table_row_index                           -                   x                   -                           -                             -
+            probability_bin                           -                   -                   x                           -                             -
+            projection_y_coordinate                   -                   -                   -                           x                             -
+            projection_x_coordinate                   -                   -                   -                           -                             x
+       Auxiliary coordinates:
+            table_row_name                            -                   x                   -                           -                             -
+       Scalar coordinates:
+            cycle_hour: 22
+            forecast_period: 68400 seconds
+       Attributes:
+            institution: Met Office
+            source: Met Office Unified Model
+            title: Reliability calibration data table
+
+The leading dimension is threshold (here shown with air_temperature thresholds).
+The reliability index dimension corresponds to the expected reliability table
```

Comment:
It's not a reliability index any more, it is a table_row_index

Reply:
Fixed and help extended slightly as discussed.